### PR TITLE
Zack auto logout

### PR DESCRIPTION
**Pull request by ZackUWA**
the functionality of auto_logout

### CODE REVIEW

File: NCARDDatabase/settings.py [hunk 1]
```diff
@@ -14,6 +14,15 @@
 import os
 import sys
 
+from datetime import timedelta
+
+AUTO_LOGOUT = {
+    'IDLE_TIME': timedelta(minutes=10),
```
Comment (id523a) [inline NCARDDatabase/settings.py:20]: Idle time should be 30 minutes

File: NCARDDatabase/settings.py [hunk 2]
```diff
@@ -14,6 +14,15 @@
...
+    # 'SESSION_TIME': timedelta(minutes=30),
```
Comment (id523a) [inline NCARDDatabase/settings.py:21]: Just delete this line instead of leaving it commented out

File: NCARDDatabase/settings.py [hunk 3]
```diff
@@ -17,7 +17,7 @@
 from datetime import timedelta
 
 AUTO_LOGOUT = {
-    'IDLE_TIME': timedelta(seconds=5),
+    'IDLE_TIME': timedelta(minutes=10),
```
Comment (id523a) [inline NCARDDatabase/settings.py:20]: Is 10 minutes the right idle time, or should it be longer? Maybe check with Alexia

File: NCARDDatabase/settings.py (hunk 2 above)
Comment (ZackUWA) [reply NCARDDatabase/settings.py:21]: delete it

**Pull request merged**